zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -151,6 +154,12 @@ public Repository(String keyspace, Cluster cluster, Boolean ensureSchema) {
                     .value("service_name", QueryBuilder.bindMarker("service_name"))
                     .using(QueryBuilder.ttl(QueryBuilder.bindMarker("ttl_"))));
 
+        selectAllSpanNames = session.prepare(
+                QueryBuilder.select("span_name")
+                        .from("span_names")
+                        .where(QueryBuilder.eq("bucket", QueryBuilder.bindMarker("bucket")))
+                        .limit(QueryBuilder.bindMarker("limit_")));
```

Comment:
was finally able to run this against cassandra in docker, and this query is outright rejected by cassandra since the span_name table primary key includes service name :( going to investigate smarter ways to go about this